pilot/cmd/pilot-discovery/server/server.go

```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/davecgh/go-spew/spew"
 	"github.com/golang/glog"
 	durpb "github.com/golang/protobuf/ptypes/duration"
-	multierror "github.com/hashicorp/go-multierror"
```

Comment:
nit: i feel like this alias is a constant source of struggle across PRs. @ZackButcher did we decide on an authoritative approach to multierror imports?

Reply:
Understood.  I only removed because it was showing up as unnecessary in GoLang.  I'm fine with whatever we decide ... especially if this is an inflammatory change :)

Reply:
Yeah, rip it out. I'm tired of having to fight the linters removing it all the time, there should never be an import alias on multierror.

Reply:
ack

Reply:
Let's not do wildcard imports; those are dangerous, and I don't feel like necessary for multierror.

Reply:
This is not a wild card import - this is removing a redundant alias.